pip/qsharp/_qsharp.py

```diff
@@ -52,6 +53,8 @@ def __init__(
 
         self._config["languageFeatures"] = language_features
         self._config["manifest"] = manifest
+        if project_root:
+            self._config["projectRoot"] = "file://" + project_root
```

Comment:
Mostly confident... the URI string we output here is for consumption of the language service, which is running in VS Code and needs to use this URI to access the corresponding folder on disk. If we are in the web, the URI would be wrong because it doesn't refer to the virtual file system... but the web + virtual file system scenario doesn't support Jupyter notebooks. If/when we have pure web hosted Jupyter we'll need to revisit this, but other things in our Python would need to change too.

Writing this up tells me I should add a comment here to capture this assumption :)

Reply:
Added comment.